packages/runtime/container-runtime/src/deltaManagerProxies.ts

```diff
@@ -194,3 +195,19 @@ export class DeltaManagerSummarizerProxy extends BaseDeltaManagerProxy {
 		this.isSummarizerClient = this.deltaManager.clientDetails.type === summarizerClientType;
 	}
 }
+
+export class DeltaManagerPendingOpsProxy extends BaseDeltaManagerProxy {
+	public get minimumSequenceNumber(): number {
+		return (
+			this.pendingStateManager.minimumPendingMessageSequenceNumber ??
```

Comment:
I do not think it should be ??. It should be min() of both (if they are present).
Consider if MSN got stuck (because of some other client not moving). This client might be at the tip of the changes, and thus its pending op can be same as last op's sequence number.
In other words, we only want to hold back MSNs, but never move that forward beyond actual MSN

Reply:
Fixed